### PR TITLE
:truck: (core) [DSDK-570]: Rename Sdk and Keyring

### PR DESCRIPTION
**Pull request by jdabbech-ledger**
### 📝 Description

Rename Keyring into Signer 
- KeyringEth => SignerEth
- KeyringBtc => SignerBtc
- Every keyring related file/folder/variable names

Rename Sdk to Dmk
- Entrypoint DeviceSDK => DeviceManagement 
- Every sdk related file/variable names 

### ❓ Context

- **JIRA or GitHub link**: [DSDK-570]

### ✅ Checklist

Pull Requests must pass CI checks and undergo code review. Set the PR as Draft if it is not yet ready for review.

- [x] **Covered by automatic tests** 
- [x] **Changeset is provided** 
- [ ] **Impact of the changes:** 
  - _list of the changes_

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.

[DSDK-570]: https://ledgerhq.atlassian.net/browse/DSDK-570?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by aussedatlo — COMMENTED** [review]
Don't forget tu update the signer ethereum readme.
Also I was thinking, we usr `btc`, `eth` but `solana` instead of `sol`. Should we plan to change this also ?

File: packages/signer/signer-btc/package.json
Comment (aussedatlo) [inline packages/signer/signer-btc/package.json:1]: [MUST] Replace the old name `@ledgerhq/keyring-btc`

File: package.json [hunk 1]
```diff
@@ -17,14 +17,14 @@
     "health-check": "turbo run health-check --output-logs=errors-only --continue",
     "core": "pnpm --filter @ledgerhq/device-management-kit",
     "context-module": "pnpm --filter @ledgerhq/context-module",
-    "keyring-btc": "pnpm --filter @ledgerhq/keyring-btc",
-    "keyring-eth": "pnpm --filter @ledgerhq/device-signer-kit-ethereum",
+    "signer-btc": "pnpm --filter @ledgerhq/signer-btc",
```
Comment (aussedatlo) [inline package.json:20]: Incoherence between `@ledgerhq/signer-btc` and `@ledgerhq/device-signer-kit-ethereum`

**Review by aussedatlo — CHANGES_REQUESTED** [review]

**Comment by jdabbech-ledger** [comment]
> Don't forget tu update the signer ethereum readme. Also I was thinking, we usr `btc`, `eth` but `solana` instead of `sol`. Should we plan to change this also ?

Yep thanks for the review.
IMO sol is more compliant indeed. Could change it in this PR if it's ok for @LedgerHQ/live-devices

File: packages/signer/signer-eth/README.md [hunk 2]
```diff
@@ -27,32 +27,32 @@ npm install @ledgerhq/device-signer-kit-ethereum
...
-The `KeyringEthBuilder.build()` method will return a `KeyringEth` instance that exposes 4 dedicated methods, each of which calls an independent use case. Each use case will return an object that contains an observable and a method called `cancel`.
+The `SignerEthBuilder.build()` method will return a `SignerEth` instance that exposes 4 dedicated methods, each of which calls an independent use case. Each use case will return an object that contains an observable and a method called `cancel`.
 
 #### Use Case 1: Get Address
 
 This method allows users to retrieve the Ethereum address according to given `derivationPath`.
 
 ```typescript
-const { observable, cancel } = keyringETH.getAddress(derivationPath, options);
+const { observable, cancel } = signerETH.getAddress(derivationPath, options);
```
Comment (aussedatlo) [inline packages/signer/signer-eth/README.md:55]: [ASK] Did you mean `SignerEth` instead of `SignerETH` ?

Comment (jdabbech-ledger) [reply packages/signer/signer-eth/README.md]: yes nice catch the variable is named `signerEth`

**Review by aussedatlo — APPROVED** [review]
LGTM 👍

File: danger/helpers.ts [hunk 3]
```diff
@@ -44,13 +44,13 @@ export const isFork = (pr: GitHubPRDSL) => pr?.head?.repo?.fork ?? false;
...
   fail: FailFn,
-  isFork: boolean = false
+  isFork: boolean = false,
 ) => ({
   regex: isFork
     ? new RegExp(`^(${BRANCH_PREFIX.join("|")})\/.+`, "i")
     : new RegExp(
-        `^(${BRANCH_PREFIX.join("|")})\/((dsdk)-[0-9]+|no-issue|issue-[0-9]+)\-.+`,
-        "i"
+        `^(${BRANCH_PREFIX.join("|")})\/((dsdk|l?dmk)-[0-9]+|no-issue|issue-[0-9]+)\-.+`,
```
Comment (mbertin-ledger) [inline danger/helpers.ts:52]: [Comment] No need as Jira project is still DSDK

File: packages/core/package.json
Comment (mbertin-ledger) [inline packages/core/package.json:1]: [MUST] Whole package could be rename from `core` to `device-managemen-kit`

**Review by mbertin-ledger — CHANGES_REQUESTED** [review]

**Review by mbertin-ledger — APPROVED** [review]

**Pull request merged**